src/content/guides/mock-network-requests.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: "../../layouts/Layout.astro"
+title: Mocking APIs and Network Requests
+description: Learn how to mock APIs and network requests in your tests (Storybook, Playwright and Cypress)
+sidebar: { order: 5, label: "Mocking APIs" }
+---
+
+# Mocking APIs and Network Requests
+
+When testing components or Pages that make network requests, it's important to mock those requests to keep your tests isolated and predictable. Chromatic integrates with functional testing frameworks to enable visual testing of your UI. Therefore, APIs are mocked by your functional testing framework, not by Chromatic itself.
```

Comment:
@winkerVSbecks small item here: why is "Pages" capitalized here?

Reply:
my bad, I'll fix it.